LineBreakpoint created at suspendFunWithInner.kt:28
Run Java
Connected to the target VM
suspendFunWithInner.kt:28
Thread stack trace:
	KotlinStackFrame b:28, SuspendFunWithInnerKt (coroutine1)
		($completion, b, bParam)
	CoroutinePreflightFrame a:21, SuspendFunWithInnerKt (coroutine1)
		($completion, $continuation, $result, a, aParam)
CoroutineInfo: -1 coroutine UNKNOWN
	CoroutineStackFrame test1:15, SuspendFunWithInnerKt (coroutine1)
		($completion, $continuation, $result, i, test1)
	CoroutineStackFrame invoke:9, SuspendFunWithInnerKt$main$result$1 (coroutine1)
		(continuation, p1, this)
	CoroutineStackFrame invokeSuspend:121, IntrinsicsKt__IntrinsicsJvmKt$createCoroutineUnintercepted$$inlined$createCoroutineFromSuspendFunction$IntrinsicsKt__IntrinsicsJvmKt$3 (kotlin.coroutines.intrinsics)
		($i$a$-createCoroutineFromSuspendFunction-IntrinsicsKt__IntrinsicsJvmKt$createCoroutineUnintercepted$2, it, result, this)
	CoroutineStackFrame resumeWith:33, BaseContinuationImpl (kotlin.coroutines.jvm.internal)
		($i$a$-with-BaseContinuationImpl$resumeWith$1, $this$with, completion, current, param, result, this)
	CoroutineStackFrame startCoroutine:128, ContinuationKt (kotlin.coroutines)
		($this$startCoroutine, completion, receiver)
	CoroutineStackFrame main:9, SuspendFunWithInnerKt (coroutine1)
		(cnt)
Disconnected from the target VM

Process finished with exit code 0
kotlin.Unit
